.github/workflows/test.yml

```diff
@@ -60,7 +70,22 @@ jobs:
         shell: bash -l {0}
         run: python -c "import numpy; numpy.show_config()"
 
+      - name: Show libraries in the system on which SciPy was built
+        shell: bash -l {0}
+        run: python -c "import scipy; scipy.show_config()"
```

Comment:
Is this something you were using for debugging, or do we want it always?

Reply:
I think it's good to include.  Scipy configs can change out from under us as dependencies update, and these things have historically caused us trouble (eg transcription velocity recently, bsseval a few years back).